content/guides/author-libraries.md

```diff
@@ -166,11 +222,29 @@ module.exports = {
 };
 ```
 
+W>At the moment of webpack 3.5.5, using the next configuration is not working properly as stated in the [issue 4824](https://github.com/webpack/webpack/issues/4824):
```

Comment:
This sentence doesn't flow nicely, how about:

"With webpack 3.5.5, using the following configuration doesn't work properly as stated in [issue 4824](https://github.com/webpack/webpack/issues/4824):"